assets/app/views/_tasks.html

```diff
@@ -19,7 +19,7 @@
             </ul>
           </div>
           <div ng-show="task.hasErrors">
-            <a href="javascript:;" ng-click="toggleExpand()">
+            <a href="javascript:;" ng-click="expanded = !expanded">
```

Comment:
`ng-repeat` creates a new copy of the scope for each loop.

https://github.com/angular/angular.js/wiki/Understanding-Scopes#ng-repeat

`expanded` is initially undefined. I can probably add back  `$scope.expanded = false` to `TaskController` if you'd rather see it initialized (untested).